acra/configuring-maintaining/general-configuration/_index.md

```diff
@@ -2,3 +2,20 @@
 title: General configuration
 bookCollapseSection: true
 ---
+
+### General configuration
+- Generate keypair (see how to do it [here]({{< ref "acra/acra-in-depth/cryptography-and-key-management/#generating-all-the-acra-keys-in-one-go-INVALID" >}})).
+- Configure secure transport using [AcraConnector]({{< ref "acra/security-controls/transport-security/acra-connector.md#acraconnector-and-acrawriter-INVALID" >}}) or [TLS]({{< ref "acra/configuring-maintaining/tls/" >}}):
+- Deploy [AcraServer]({{< ref "acra/configuring-maintaining/general-configuration/acra-server.md#getting-started-with-acraserver-INVALID" >}}) to a separate machine:
+- Integrate AcraWriter closely following the [examples](https://github.com/cossacklabs/acra/tree/master/examples) and using either AcraConnector (if available on this machine) or direct TLS database connection.
+- Test everything:
+
+  **1.** Connect AcraConnector to AcraServer, send a regular request to your database through AcraConnector.<br/>_Result_: If you see the answer, AcraConnector and AcraWriter are able to connect and forward signals back and forth. It means that the network and the keys are fine.
+
+  **2.** Upon integrating AcraWriter into your code, try generating an [AcraStruct]({{< ref "acra/acra-in-depth/data-structures/#understanding-acrastruct" >}}) from some payload.<br/>_Result_: If you succeed in running AcraWriter code, Themis library is installed properly and some of the keys are located in the expected place.
+
+  **3.** Write a row with AcraStruct into the database, either directly or through AcraConnector. Request this row through AcraConnector.<br/>_Result_: If you see decrypted payload in the response, the scheme works properly.
```

Comment:
I believe it would lead to overall better result if ordered lists are represented with ordered lists. You can use CSS to make numbers bold.

```markdown
  1. Connect AcraConnector to AcraServer, send a regular request to your database through AcraConnector.

     _Result_: If you see the answer, AcraConnector and AcraWriter are able to connect and forward signals back and forth.
     It means that the network and the keys are fine.

  2. Upon integrating AcraWriter into your code, try generating an [AcraStruct] from some payload.

     _Result_: If you succeed in running AcraWriter code,
     Themis library is installed properly and some of the keys are located in the expected place.

  3. Write a row with AcraStruct into the database, either directly or through AcraConnector.
     Request this row through AcraConnector.

     _Result_: If you see decrypted payload in the response, the scheme works properly.

[AcraStruct]: {{< ref "acra/acra-in-depth/data-structures/#understanding-acrastruct" >}}
```

Here and in other places.

Reply:
My point is that if you write things like this:

```markdown
  **1.** One
  **2.** Two
  **3.** Three
```
then it will not render into `<ol>`, as opposed to this:
```markdown
  1. One
  2. Two
  3. Three
```
The latter form should be preferred. If the number absolutely must be bold, `ol::marker` in CSS will take care of that, globally, across all documentation.